monitor/flowgen/util.go

```diff
@@ -0,0 +1,16 @@
+package flowgen
+
+import (
+	"math/big"
+
+	"github.com/omni-network/omni/lib/tokens"
+)
+
+func spend(token tokens.Token, amt float64) Spend {
+	return Spend{token: unit(amt, token.Decimals)}
+}
+
+func unit(amt float64, decimals uint) *big.Int {
```

Comment:
please write some unit tests for this

Reply:
can also use `math.Pow10`

also, maybe panic if `g < 1`

Reply:
yeah will add

Reply:
just skelotonning